src/System.Private.CoreLib/src/System/ThrowHelper.cs

```diff
@@ -298,32 +298,17 @@ internal static void ThrowArraySegmentCtorValidationFailedExceptions(Array array
 
         internal static void ThrowFormatException_BadFormatSpecifier()
         {
-            throw CreateFormatException_BadFormatSpecifier();
-        }
-
-        private static Exception CreateFormatException_BadFormatSpecifier()
```

Comment:
The split into separate methods to throw the exception and a small method to create the exception was needed to workaround issues in old .NET Framework JITs. It is not necessary in .NET Core. We do not have any other `CreateXXX` method in the CoreLib throwhelper.

Reply:
We do have a lot of `GetXXX` methods though, do they fall into the same category?

Reply:
The `GetXXX` method make sense once the throw helper becomes complex and the JIT would have troubles to see that the method always throws. `GetArraySegmentCtorValidationFailedException` is a good example. 

A few of the existing `GetXXX` may not be required, but it is hard to tell for sure by just looking at them.